libminifi/test/integration/StateTransactionalityTests.cpp

```diff
@@ -72,8 +72,7 @@ class StatefulIntegrationTest : public IntegrationBase {
         // set hooks
         const auto processController = dynamic_cast<ProcessorController*>(&component);
         assert(processController != nullptr);
-        stateful_processor_ = dynamic_cast<StatefulProcessor*>(processController->getProcessor());
-        assert(stateful_processor_ != nullptr);
```

Comment:
why isn't this assert needed anymore?

Reply:
The return type of getProcessor is a reference now, it's always present.